float32 r11
float32 r12
float32 r13
float32 r14
float32 r21
float32 r22
float32 r23
float32 r24
float32 r31
float32 r32
float32 r33
float32 r34
float32 gripper
bool rotate
bool rest_pos
---
int64 Success
float32 gripper_fb
